WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/Blaze/DashboardBlazeCardCell.swift

```diff
@@ -55,7 +55,7 @@ class DashboardBlazeCardCell: DashboardCollectionViewCell {
 
     private func setupView() {
         contentView.addSubview(cardView)
-        contentView.pinSubviewToAllEdges(cardView, priority: .defaultHigh)
+        contentView.pinSubviewToAllEdges(cardView, priority: UILayoutPriority(999))
```

Comment:
I used 999 because it's embedded in the collection view cell, and `.required` may trigger auto layout warnings during the size calculation. `.defaultHigh` (750) was too low because it matches the compression resistance of the text, making the layout ambiguous.